src/modules/fortune-admin/controllers.js

```diff
@@ -2,6 +2,35 @@
 angular.module('fortuneAdmin.Controllers', [
     'fortuneAdmin.Services'
   ])
+  .controller('faActionsCtrl', [
+    '$scope', 
+    'faActionsService',
+    '$sce',
+    '$modal',
+    function ($scope, faActionsService, $sce, $modal) {
+      $scope.actions = faActionsService.getActions('users');
+      $scope.selected = $scope.actions[1];
+      $scope.isModalVisible = false;
+  
+      $scope.selectAction = function (action) {
+        $scope.isModalVisible = action.type === 'modal';
+        console.log($scope.entity);
+        if($scope.isModalVisible && action.createTpl) {
+           $modal.open({
+             template : $sce.trustAsHtml(action.createTpl($scope.entity)),
```

Comment:
What's the point of using `$sce` there? Cannot figure out - `$modal` expects html already, or at least i would expect it to.